app/javascript/components/supplemental_application/sections/RentalAssistance.js

```diff
@@ -165,85 +169,94 @@ const RentalAssistanceForm = ({ values, onSave, loading, onClose, applicationMem
-            />
-          </FormGrid.Item>
-        </FormGrid.Row>
-        {isOther(form.getState().values.rental_assistances[index]) && (
-          <FormGrid.Row expand={false}>
+        <InlineModal whiteBackground marginBottom={isNew} >
+          <FormGrid.Row>
+            <FormGrid.Item>
+              <SelectField
+                label='Recipient'
+                fieldName={`rental_assistances.${index}.recipient`}
+                options={applicationMembersOptions}
+                className='rental-assistance-recipient'
+              />
+            </FormGrid.Item>
+          </FormGrid.Row>
+          <FormGrid.Row>
             <FormGrid.Item>
-              <InputField
-                label='Other Assistance Name'
-                fieldName={`rental_assistances.${index}.other_assistance_name`} />
+              <SelectField
+                label='Type of Assistance'
+                fieldName={`rental_assistances.${index}.type_of_assistance`}
+                options={typeOfAssistanceOptions}
+                className='rental-assistance-type'
+                validation={validate.isPresent('Please select a type of assistance.')}
+              />
             </FormGrid.Item>
           </FormGrid.Row>
-        )}
-        <FormGrid.Row expand={false}>
-          <div className='form-grid_item column'>
-            <button
-              className='button primary tiny margin-right margin-bottom-none'
-              type='button'
-              onClick={onSaveWithIndex}
-              disabled={loading}>
-              Save
-            </button>
-            <button
-              className='button secondary tiny margin-right margin-bottom-none'
-              type='button'
-              onClick={onClose}
-              disabled={loading}>
-              Cancel
-            </button>
-            {!isNew && (
-              <button
-                className='button alert-fill tiny margin-bottom-none right'
-                type='button'
-                onClick={onDelete}
-                disabled={loading}>
-                Delete
-              </button>
-            )}
-          </div>
-        </FormGrid.Row>
+          <FormGrid.Row>
+            <FormGrid.Item>
+              <CurrencyField
+                label='Assistance Amount'
+                fieldName={`rental_assistances.${index}.assistance_amount`}
+                validation={validateAssistanceAmount}
+                id='assistance_amount'
+                isDirty={visited && visited[`rental_assistances.${index}.assistance_amount`]}
+              />
+            </FormGrid.Item>
+          </FormGrid.Row>
+          <FormGrid.Row>
+            <FormGrid.Item>
+              <YesNoRadioGroup
+                label='Recurring Assistance'
+                fieldName={`rental_assistances.${index}.recurring_assistance`}
+                uniqId={(!isEmpty(values) && values.id) || 'new'}
+                trueValue='Yes'
```

Comment:
I think these are the defaults. You're welcome to keep them if you'd prefer to be explicit though

Reply:
actually looks like true/falseLabel are yes/no but not true/falseValue. Snapshots caught something helpful for once!